src/mpi/info/info_set.c

```diff
@@ -125,7 +122,6 @@ int MPI_Info_set(MPI_Info info, const char *key, const char *value)
     mpi_errno = MPIR_Err_return_comm(NULL, __func__, mpi_errno);
     goto fn_exit;
     /* --END ERROR HANDLING-- */
-#endif
 }
```

Comment:
Just for my better understanding ... Why is the way we handle errors here different from other parts of the code? For example, in https://github.com/pmodels/mpich/blob/f73ecd6f12558d241eecf6e3986abb14d7cf2b29/src/mpi/pt2pt/send.c#L156 we don't call `MPIR_Err_create_code` unless we have error checking.

Reply:
That's a good point. I think the model in send is a better one to copy. I have updated the PR to match it.